### ISSUE TITLE
Bed with wrong color is dropped when broken from the bottom part

### ISSUE THREAD

**Issue by Leo3418** [GenisysPro/GenisysPro#706]
#### Issue description
Place a bed in any color other than red, then break it from the bottom part (the side against the "pillow"). The dropped item is always in red, no matter what the original color of the bed is.  
However, if the bed is broken from another part, the dropped item is in the correct color.

#### Platform information
* Git commit: cb61a69

* PHP binary version: 7.0.21

* Server OS: Windows 10 1703 x64

* Game client: Minecraft: Windows 10 Edition v1.1.5

#### Crash dump, back trace, memory dumps, plugins list or other files (if any)
![ezgif com-gif-maker](https://user-images.githubusercontent.com/14175175/28996633-17d9d940-7a37-11e7-8c85-80b16f656d6f.gif)

**Comment by DerddyDert** [comment]
Yeah, the bottom block is different... But what a beautiful example of what an issue report is supposed to look like!

**Comment by xISRAPILx** [comment]
LOOOOOOOL

**Comment by DerddyDert** [comment]
I think its because my feet were so dirty it made all the beds red at the bottom :-o

**Comment by chadigos** [comment]
the second half is unknown block ,so getColor false

**Comment by Leo3418** [comment]
@chadigos Yeah, it's `NULL`

**Comment by GamakCZ** [comment]
I think it's bug minecraft, in classical survival it happened to me that I destroyed the bed and nothing went wrong with it.
